<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDeckDialog</class>
 <widget class="QDialog" name="DeleteDeckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/img/img/menuBg.png)</string>
  </property>
  <widget class="QPushButton" name="deck4Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(197, 197, 197);</string>
   </property>
   <property name="text">
    <string>Deck4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>466</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190, 190, 190)</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deck5Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(197, 197, 197);</string>
   </property>
   <property name="text">
    <string>Deck5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deck2Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(197, 197, 197);</string>
   </property>
   <property name="text">
    <string>Deck2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deck3Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(197, 197, 197);</string>
   </property>
   <property name="text">
    <string>Deck3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:Transparent;
color:rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline; color:#ffffff;&quot;&gt;Suppression d'un deck&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deck1Button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(197, 197, 197);</string>
   </property>
   <property name="text">
    <string>Deck1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
